apis/fluent-en_GB/atrium-api-fluent-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/pathAssertions.kt

```diff
@@ -379,3 +379,14 @@ fun <T : Path> Expect<T>.hasSameTextualContentAs(
  */
 fun <T : Path> Expect<T>.hasSameBinaryContentAs(targetPath: Path): Expect<T> =
     _logicAppend { hasSameBinaryContentAs(targetPath) }
+
+/**
+ * Expects that the subject of `this` expectation (a [Path]) is an empty directory;
+ * meaning that there is a file system entry at the location the [Path] points to and that is an empty directory.
+ *
+ * @return an [Expect] for the subject of `this` expectation.
+ *
+ * @since 0.16.0
```

Comment:
Please add a sample to PathAssertionSamples and link it here via `@sample` (see for instance anyAssertions.kt)
Could you do me a favour and also link toBeASymbolicLink with the corresponding sample.